gui/projectfiledialog.cpp

```diff
@@ -945,11 +945,10 @@ void ProjectFileDialog::editSuppression(const QModelIndex & /*index*/)
 int ProjectFileDialog::getSuppressionIndex(const QString &shortText) const
 {
     const std::string s = shortText.toStdString();
-    for (int i = 0; i < mSuppressions.size(); ++i) {
-        if (mSuppressions[i].getText() == s)
-            return i;
-    }
-    return -1;
+    auto it = std::find_if(mSuppressions.begin(), mSuppressions.end(), [&](const SuppressionList::Suppression& sup) {
```

Comment:
pedantic nit (for the future): I would prefer if we use `cbegin()` and `cend()` even when the context is `const`.